creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -29,8 +33,39 @@ const respondToURSMRequestForProposalController = async (req) => {
   }
 }
 
+/**
+ * Given walletPublicKeys array and target creatorNodeEndpoint, will request export
+ * of all user data, update DB state accordingly, fetch all files and make them available.
+ *
+ * This route is only run on secondaries, to export and sync data from a user's primary.
+ */
+const syncRouteController = async (req, res) => {
```

Comment:
all code from lines 43-60 is direct copy of original /sync route code in `nodesync.js`. only change is to call `enqueueSync()` on line 61 instead